### PR TITLE
Add support for upstream Stein release

### PR DESCRIPTION
**Pull request by rkukura**
Depend on stable/stein rather than stable/rocky branch of upstream
Neutron repository. Changes needed for compatability with stable/stein
that were also compatible with stable/rocky were made in previous
patches, so only Stein-specific changes are included here.

### CODE REVIEW

**Comment by rkukura** [comment]
This needs significant work before merging, primarily to deal with upstream Neutron moving in Stein from using ovs-vsctl commands to using ovsdb.

File: test-requirements.txt [hunk 1]
```diff
@@ -3,8 +3,9 @@
 # process, which may cause wedges in the gate later.
 hacking>=1.1.0,<1.2.0 # Apache-2.0
 
--e git+https://opendev.org/openstack/neutron.git@stable/rocky#egg=neutron
+neutron
 
+contextlib2>=0.5.4
```
Comment (kentwu1) [inline test-requirements.txt:8]: I'm wondering where this is from? My understanding is that this is based on this list,
https://github.com/openstack/neutron/blob/stable/stein/test-requirements.txt
but I didn't find contextlib2 there so.....

File: opflexagent/utils/bridge_managers/ovs_lib.py [hunk 2]
```diff
@@ -13,7 +13,9 @@
 import contextlib
 
 from neutron.agent.common import ovs_lib
-from neutron.agent.ovsdb import impl_vsctl
+
+# REVISIT: This is gone in stein!
```
Comment (kentwu1) [inline opflexagent/utils/bridge_managers/ovs_lib.py:17]: Is there a reason we still keep this around?

File: opflexagent/utils/bridge_managers/ovs_lib.py [hunk 3]
```diff
@@ -26,9 +28,11 @@ def __init__(self, *args, **kwargs):
         super(OVSBridge, self).__init__(*args, **kwargs)
 
     def reset_ofversion(self):
-        context = self.ovsdb.context
-        return impl_vsctl.BaseCommand(context, 'set',
-                                      args=[self.br_name, 'protocols=[]'])
+        # REVISIT: impl_vsctl is gone in stein!
```
Comment (kentwu1) [inline opflexagent/utils/bridge_managers/ovs_lib.py:31]: ditto

File: test-requirements.txt (hunk 1 above)
Comment (tbachman) [reply test-requirements.txt:8]: I had to add this - the python35 job was failing. I've seen references to it not being installed, even though it's considered a standard library. Adding this fixed the python35 job.

File: opflexagent/utils/bridge_managers/ovs_lib.py (hunk 3 above)
Comment (tbachman) [reply opflexagent/utils/bridge_managers/ovs_lib.py:31]: will fix (along with the other one I accidentally resolved).

File: opflexagent/utils/bridge_managers/ovs_lib.py (hunk 2 above)
Comment (tbachman) [reply opflexagent/utils/bridge_managers/ovs_lib.py:17]: Will fix.

**Comment by tbachman** [comment]
Pushed update, addressing reviewer comments.

**Pull request merged**